keps/420-partial-admission/README.md

```diff
@@ -144,6 +155,21 @@ In case of MPIJob `j.Spec.RunPolicy.SchedulingPolicy.MinAvailable` can be used t
 Whether an MPIJob supports partial admission or not can be deduced based on `MinAvailable` without the need of a dedicated annotation.
 Additional research is needed into the potential usage of multiple variable count PodSets.
 
+### kubernetes-sigs/jobset
+
+Besides adapting `RunWithPodSetsInfo` and `RestorePodSetsInfo` it should also:
+
+- rework `PodSets()` to populate `MinCount` and `PartialAdmissionDecrementStep` if the jobset is marked to support partial admission.
+  * jobsets supporting partial admission should have a dedicated annotation. eg. `kueue.x-k8s.io/jobset-min-replicas`, indicating the index of the replicated job that supports partial admission as well as the minimum `replicas` acceptable.
+
+```
+MinCount = minimumReplicas * replicatedJobParallelism
+PartialAdmissionDecrementStep = replicatedJobParallelism
+```
+
+NOTE: Currently jobset is not supporting `replicas` updates, this needs to be implemented on the jobset side.
```

Comment:
Can you post this on Jobset repo?

Reply:
If we are going ahead with this, I'll open an issue on jobset side and try to provide the implementation.